### PR TITLE
[REVIEW] Upload rmm package per version python-cuda combo

### PR DESCRIPTION
**Pull request by galipremsagar**
This PR enables uploading rmm package per python-cuda version combo.

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: ci/cpu/prebuild.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-#Build rmm once per PYTHON
-if [[ "$CUDA" == "10.1" ]]; then
+#Build rmm once per PYTHON-CUDA combo
+if [[ "$PYTHON" == "3.7" ]] && [[ "$CUDA" == "10.1" ]]; then
     export UPLOAD_RMM=1
 else
     export UPLOAD_RMM=0
```
Comment (kkraus14) [inline ci/cpu/prebuild.sh:7]: This should just always set `UPLOAD_RMM=1`, you don't want to only upload when Python is 3.7 and CUDA is 10.1

File: ci/cpu/prebuild.sh [hunk 2]
```diff
@@ -1,11 +1,7 @@
 #!/usr/bin/env bash
 
-#Build rmm once per PYTHON
-if [[ "$CUDA" == "10.1" ]]; then
-    export UPLOAD_RMM=1
-else
-    export UPLOAD_RMM=0
-fi
+#Build rmm once per PYTHON-CUDA combo
```
Comment (kkraus14) [inline ci/cpu/prebuild.sh:3]:
```suggestion
#Always upload RMM Python package
```

File: conda/recipes/rmm/meta.yaml [hunk 3]
```diff
@@ -1,7 +1,9 @@
-# Copyright (c) 2019, NVIDIA CORPORATION.
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
 
 {% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
 {% set py_version=environ.get('CONDA_PY', 35) %}
+{% set cuda_version='.'.join(environ.get('CUDA_VERSION', '10.1').split('.')[:2]) %}
```
Comment (harrism) [inline conda/recipes/rmm/meta.yaml:5]: Is this hard-coding 10.1?

Comment (kkraus14) [reply conda/recipes/rmm/meta.yaml:5]: No, this reads the environment variable `CUDA_VERSION` into a jinja variable which can be used in jinja expressions below

Comment (harrism) [reply conda/recipes/rmm/meta.yaml:5]: So what is the 10.1?

Comment (kkraus14) [reply conda/recipes/rmm/meta.yaml:5]: > So what is the 10.1?

The default if the environment variable doesn't exist.

**Pull request merged**